SixTest/stf_output/compare.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+cd stf/
+gfortran new_converter.f
```

Comment:
FYI: When running the compilers like gfortran or gcc, use the `-o filename` flag to set the name of the executable, so it gets called something more informative than `a.out`. Also use the `-Wall` flag.